### PR TITLE
introduce experimental multi-site and OIDC hidden flags to tctl and tsh

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: lib/services/local/users.go [hunk 1]
```diff
@@ -380,6 +380,9 @@ func (s *IdentityService) DeleteOIDCConnector(connectorID string) error {
 func (s *IdentityService) GetOIDCConnector(id string, withSecrets bool) (*services.OIDCConnector, error) {
 	out, err := s.backend.GetVal(connectorsPath, id)
 	if err != nil {
+		if teleport.IsNotFound(err) {
+			return nil, trace.Wrap(teleport.NotFound(fmt.Sprintf("OpenID Connect connector '%v' not found", id)))
```
Comment (kontsevoy) [inline lib/services/local/users.go]: Is this a user-facing error? "Connector not found" does not tell me what to do. What is the most likely cause for seeing this? Configuration error? Perhaps we should say "OpenID connector 'xxx' is not supported?"

File: tool/tctl/main.go [hunk 2]
```diff
@@ -113,6 +114,7 @@ func main() {
 	userAdd.Arg("login", "Teleport user login").Required().StringVar(&cmdUsers.login)
 	userAdd.Arg("local-logins", "Local UNIX users this account can log in as [login]").
 		Default("").StringVar(&cmdUsers.allowedLogins)
+	userAdd.Flag("identity", "[EXPERIMENTAL] Add OpenID Connect identity, e.g. --identity=google:bob@gmail.com").Hidden().StringsVar(&cmdUsers.identities)
```
Comment (kontsevoy) [inline tool/tctl/main.go]: nice!

**Comment by kontsevoy** [comment]
lgtm but see the error question above.

File: lib/services/local/users.go (hunk 1 above)
Comment (klizhentas) [inline lib/services/local/users.go]: k, I'll update the message

**Comment by klizhentas** [comment]
updated error message and merged

**Pull request merged**